pkg/skaffold/initializer/build/cli.go

```diff
@@ -62,29 +63,32 @@ func (c *cliBuildInitializer) GenerateManifests() (map[GeneratedBuilderImagePair
 }
 
 func (c *cliBuildInitializer) processCliArtifacts() error {
-	pairs, err := processCliArtifacts(c.cliArtifacts)
+	pairs, workspaces, err := processCliArtifacts(c.cliArtifacts)
 	if err != nil {
 		return err
 	}
 	c.builderImagePairs = pairs
+	c.workspaces = workspaces
 	return nil
 }
 
-func processCliArtifacts(cliArtifacts []string) ([]BuilderImagePair, error) {
+func processCliArtifacts(cliArtifacts []string) ([]BuilderImagePair, []string, error) {
 	var pairs []BuilderImagePair
+	var workspaces []string
 	for _, artifact := range cliArtifacts {
-		// Parses JSON in the form of: {"builder":"Name of Builder","payload":{...},"image":"image.name"}.
+		// Parses JSON in the form of: {"builder":"Name of Builder","payload":{...},"image":"image.name","context":"artifact.context"}.
 		// The builder field is parsed first to determine the builder type, and the payload is parsed
 		// afterwards once the type is determined.
```

Comment:
It would be helpful if we described the older format too.  Something like:
```
// `--artifacts` supports two forms of specifying artifacts:
//   1. A key-value pair: `path/to/Dockerfile=imageName` (deprecated, historical, Docker-only) 
//   2. JSON object: ...

Reply:
done!